.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,48 +3,35 @@ Resolves
 
 ## Submitter Checklist:
 
-- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
+- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
 - [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
-- [ ] Added/updated tests for this change (for new code or code which already has tests).
-- Verified that these changes build without errors on
-  - [ ] Android
-  - [ ] iOS
-  - [ ] Linux
-  - [ ] macOS
-  - [ ] Windows
-- Verified that these changes pass automated tests (unit, browser, security tests) on
-  - [ ] iOS
-  - [ ] Linux
-  - [ ] macOS
-  - [ ] Windows
-- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`, `npm run gn_check`)
+- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
+- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`
 - [ ] Ran `git rebase master` (if needed).
```

Comment:
Can we do something similar to the first checkbox - describing the state of the PR rather than an "if needed" action? I'm thinking something along the lines of:
> - [ ] The PR is rebased/up to date with the most recent master branch

Reply:
I believe this state is nearly impossible - master is frequently updated, so in a minute after rebase you are not up-to-date anymore. So the current sentence just reminds that it's useful to rebase once in a while.

And if your RP is seriously outdated, CI wouldn't merge it anyway